sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/Tests/DnsForwardingRulesetTests.cs

```diff
@@ -16,49 +16,45 @@ public class DnsForwardingRulesetTests : DnsResolverTestBase
         private DnsResolverResource _dnsResolver;
         private DnsForwardingRulesetCollection _dnsForwardingRulesetCollection;
         private ResourceIdentifier _outboundEndpointId;
-        private string _vnetId;
-        private string _subnetId;
+        //private string _vnetId ;
+        //private string _subnetId;
 
-        public DnsForwardingRulesetTests(bool async) : base(async)
+        public DnsForwardingRulesetTests(bool async) : base(async)//, RecordedTestMode.Record)
         {
         }
 
-        [SetUp]
         public async Task CreateDnsResolverCollection()
         {
             var dnsResolverName = Recording.GenerateAssetName("dnsResolver-");
             var vnetName = Recording.GenerateAssetName("vnet-");
+            var resourceGroup = await CreateResourceGroupAsync();
+            var outboundEndpointName = Recording.GenerateAssetName("outboundEndpoint-");
 
-            _vnetId = $"/subscriptions/{TestEnvironment.SubscriptionId}/resourceGroups/{TestEnvironment.ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}";
-            _subnetId = $"/subscriptions/{TestEnvironment.SubscriptionId}/resourceGroups/{TestEnvironment.ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{SubnetName}";
-
-            var dnsResolverData = new DnsResolverData(this.DefaultLocation, new WritableSubResource
-            {
-                Id = new ResourceIdentifier(_vnetId)
-            });
+            //_vnetId = $"/subscriptions/{TestEnvironment.SubscriptionId}/resourceGroups/{TestEnvironment.ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}";
+            //_subnetId = $"/subscriptions/{TestEnvironment.SubscriptionId}/resourceGroups/{TestEnvironment.ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{SubnetName}";
 
             if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
             {
-                await CreateVirtualNetworkAsync(vnetName);
+                await CreateVirtualNetworkAsync();
             }
```

Comment:
If we have changed the method signature of `CreateVirtualNetworkAsync`, we could do this instead:
```suggestion
(var vnetId, var subnetId) = await CreateVirtualNetworkAsync();
```
and then we could use these as local variables in this method to make the process more clear

Reply:
fixed